examples/basic_bitcoin/test/tests.ts

```diff
@@ -37,15 +37,15 @@ export function getTests(canisterId: string): Test[] {
         {
```

Comment:
See if you can make all of the tests across the two examples the same. If not, let's make sure psbt is well-tested to similar basic_bitcoin levels

Reply:
duplicate